Eventy/src/app/infrastructure/auth/register-organizer/register-organizer.component.ts

```diff
@@ -52,7 +54,24 @@ export class RegisterOrganizerComponent {
     } else {
       let user: RegisterData = this.registerForm.value as RegisterData;
       user.profilePictures = [this.registerForm.controls['profilePicture'].value];
-      this.authService.register(user);
+      this.authService.register(user).subscribe({
+        next: (response: string) => {this.dialog.open(InvalidInputDataDialogComponent, {
+          data : {
+            title: "Confirmation needed!",
+            message: "Confirmation email has been sent to you"
+          }
+        });
```

Comment:
NOTE: you can use my custom dialog here, it is more fancy!!!

Reply:
Definitely will check this out later!! Your dialog is much better, but right now I just want to make this work! ✨